SDK/AppCenter/Microsoft.AppCenter.UWP/Utils/DeviceInformationHelper.cs

```diff
@@ -14,17 +14,14 @@ namespace Microsoft.AppCenter.Utils
     /// </summary>
     public class DeviceInformationHelper : AbstractDeviceInformationHelper
     {
-        public static event EventHandler InformationInvalidated;
-        private static string _country;
         private static IScreenSizeProvider ScreenSizeProvider;
         private static IScreenSizeProviderFactory ScreenSizeProviderFactory =
             new DefaultScreenSizeProviderFactory();
 
-        // This method must be called *before* any instance of DeviceInformationHelper has been created
```

Comment:
This comment is still true because there could now be a null reference exception if it's called later

Reply:
Why? We have default value in two line above + it's only for tests, right?

Reply:
Yeah not a big deal but if you set screen provider factory it could cause a crash because an instance that's already created could expect the value for screensizeprovider to be nonnull.